src/tsomi.js

```diff
@@ -66,7 +66,12 @@ var dateFormat = d3.time.format('%Y-%m-%d');
 $(".search-input").keyup(function (e) {
   if (e.keyCode == 13) {
     searchForPeople(e.target.value, function(results) {
-      console.log("results", results);
+        if (results.length > 0) {
+            var subject = convertSpaces(results[0].name.value);
+            location.href = "/?subject=" + subject;
```

Comment:
this will reload the whole page on search. i think we'd like to avoid that, if possible.

let's look into https://developer.mozilla.org/en-US/docs/Web/API/History to change the URL, and https://github.com/trebor/tsomi/blob/master/src/components/Renderer.js#L407 for an example of how to rerender with a new subject.

thanks, pam!

Reply:
Thanks!  Will look at that